torch/csrc/distributed/autograd/functions/SendRpcBackwards.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <torch/csrc/autograd/function.h>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+struct TORCH_API SendRpcBackwards : public torch::autograd::Node {
+  torch::autograd::variable_list apply(
+      torch::autograd::variable_list&& grads) override;
+};
+
```

Comment:
We would also need a `RecvRpcBackward` to pair with this `SendRpcBackward`, no?

Reply:
Yes, I wanted to keep this PR simple and added only `send` for early feedback on the general design. Will follow up on `recv` in a different PR.